neurons/validators/validator.py

```diff
@@ -172,7 +173,20 @@ def get_reward(self, response: DiscoveryOutput, ip_per_hotkey=None, run_id_per_h
         )
 
         if self.validator_config.is_grace_period and response.version == 5:
```

Comment:
and why we have here version 5? version 5 is the new one, so it should be 4 ? @0xMarx